### PR TITLE
nvc 1.7.1

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/nvc.rb [hunk 1]
```diff
@@ -31,23 +31,15 @@ class Nvc < Formula
 
   resource "homebrew-test" do
     url "https://github.com/suoto/vim-hdl-examples.git",
+        branch:   "master",
```
Comment (carlocab) [inline Formula/nvc.rb:34]: This isn't needed, since we specify `revision`.

File: Formula/nvc.rb [hunk 2]
```diff
@@ -31,23 +31,15 @@ class Nvc < Formula
...
-      sha256 "d5bda0f89c346f618b9bc5ce96095be5bb9eb8e0fec3caea4ebddfe1ae2dee23"
-    end
-  end
-
   def install
     system "./autogen.sh" if build.head?
 
     # Avoid hardcoding path to the `ld` shim.
-    inreplace "configure", "#define LINKER_PATH \\\"$linker_path\\\"", "#define LINKER_PATH \\\"ld\\\"" if OS.linux?
+    inreplace "configure", "#define LINKER_PATH \"$linker_path\"", "#define LINKER_PATH \"ld\"" if OS.linux?
```
Comment (carlocab) [inline Formula/nvc.rb:42]: Does passing `linker_path` to `configure` instead of an `inreplace` work?

Comment (carlocab) [reply Formula/nvc.rb:42]: Actually, we can probably set `ac_cv_path_linker_path`.

Comment (carlocab) [reply Formula/nvc.rb:42]: Let's try it.

**Review by carlocab — APPROVED** [review]

File: Formula/nvc.rb (hunk 1 above)
Comment (alebcay) [reply Formula/nvc.rb]: Interesting, `audit` complained to me about the lack of `branch` on Linux.

**Pull request closed without merging**